samples/boards/st/power_mgmt/wkup_pins/boards/nucleo_f302r8.overlay

```diff
@@ -0,0 +1,21 @@
+/ {
```

Comment:
Could you add a copyright/license notice inline comment as requested by CI compliance checks?

Reply:
Fixed.